SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -167,6 +167,18 @@ bool USpatialActorChannel::IsDynamicArrayHandle(UObject* Object, uint16 Handle)
 	return RepLayout->Cmds[RepLayout->BaseHandleToCmdIndex[Handle - 1].CmdIndex].Type == ERepLayoutCmdType::DynamicArray;
 }
 
+void USpatialActorChannel::UpdateShadowData()
+{
+	FObjectReplicator& ActorObjReplicator = FindOrCreateReplicator(Actor).Get();
+	ActorObjReplicator.RepLayout->InitShadowData(ActorObjReplicator.ChangelistMgr->GetRepChangelistState()->StaticBuffer, Actor->GetClass(), (uint8*)Actor);
```

Comment:
```suggestion
	ActorReplicator->RepLayout->InitShadowData(ActorReplicator->ChangelistMgr->GetRepChangelistState()->StaticBuffer, Actor->GetClass(), (uint8*)Actor);
```
Actor's replicator is already cached in the `ActorReplicator` variable.